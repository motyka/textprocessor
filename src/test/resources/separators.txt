first new line
second! separated. by, a dot
third separated, by a punctuation
forth separated by a space
fifthIsToLongAndForcedToSeparateAtTheLimit