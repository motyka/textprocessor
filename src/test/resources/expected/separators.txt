[{text=first new line, start=0, length=14, contains=false}, {text=second! separated., start=16, length=18, contains=true}, {text=by, a dot, start=35, length=9, contains=false}, {text=third separated,, start=46, length=16, contains=true}, {text=by a punctuation, start=63, length=16, contains=false}, {text=forth separated, start=81, length=15, contains=true}, {text=by a space, start=97, length=10, contains=false}, {text=fifthIsToLongAndForc, start=109, length=20, contains=false}, {text=edToSeparateAtTheLim, start=129, length=20, contains=false}, {text=it, start=149, length=2, contains=false}]